packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -3,83 +3,79 @@ id: Testing your accessibility
 ---
 
-## General testing
+# Testing your product's accessibility 
 
-Many accessibility issues can be found by doing a few simple checks: 
-
-1. **Validate your HTML.** Structural, semantic HTML is the key starting point toward good accessibility practices. When a screen reader (or any sort of assistive device) scans a web page, it gets information about the Document Object Model (DOM), or the HTML structure of the page. No styles or JavaScript will be read by a screen reader. 
-
-   - Screen reader software like Voice Over, NVDA, or JAWS doesn’t just turn text into speech. It can use information in the HTML to list all of the headings on a page, give extra navigation controls to data tables, or announce how many items are in a list, among other things. This makes semantic HTML essential. 
-
-   - There are many tools you can use to validate your HTML, such as [W3C’s markup validation service](https://validator.w3.org/). 
-
-
-2. **Use an accessibility audit tool to check for violations.** If you are using PatternFly in your project, we recommend using [aXe: The Accessibility Engine](//www.deque.com/axe/) to check for accessibility violations locally. To make it even easier for you, we've created the [patternfly-a11y script](https://github.com/patternfly/patternfly-a11y) which reports any axe accessibility violations from a set of pages. A configuration file can be set to customize to specific needs like authentication, waiting for any specific items to finish loading (like a loading spinner), etc. When used as a report, it will give an output [similar to this example](http://a11y-os.surge.sh/). The UI should be built first in the CI workflow and then a job created to run the a11y script against that build. The a11y script assumes that there is a webserver running somewhere that is serving up the pages, i.e. in localhost:8000, that it can reach. If you want to test a step before deployment, you could follow the path of integrating axe with cypress. If you are contributing to PatternFly, refer to our [README.md](//github.com/patternfly/patternfly/blob/main/README.md#testing-for-accessibility) on how to run this tool.
-
-
-3. **Test keyboard accessibility, and check that these requirements are met:**
-    - All functionality is keyboard accessible.
-    - Elements in the HTML and in the layout follow a logical order.
-    - Elements with focus are clearly visible.
+The following guide contains instructions and recommendations to comprehensively test your product's accessibility to identify accessibility issues and opportunities for improvement.  
 
+## General testing recommendations 
 
-4. **Disable styles**, then test the information architecture and presence of adequate text labels. The [WAVE browser extension from WebAIM](//wave.webaim.org/extension/) provides this feature if it isn't available in the browser you are using.
+Many accessibility issues can be found by doing a few simple checks: 
 
+### 1. Validate your HTML
```

Comment:
Yeah - I'll update that! I was 50/50 on doing this & mostly wanted to visualize what it'd look like, but since you mentioned it I think that's good confirmation to just keep normal headings ty 😊